### PR TITLE
(maint) Minor cleanup of the error reporting in the Ruby API

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

File: ruby/src/api.cc [hunk 1]
```diff
@@ -469,7 +469,7 @@ namespace leatherman { namespace ruby {
                                                                   "break file if File.exist? file;"
                                                                   "false end)" });
         if (!success) {
-            LOG_WARNING("ruby failed to run", output, none);
+            LOG_WARNING("ruby failed to run");
```
Comment (MikaelSmith) [inline ruby/src/api.cc]: Comparing te [stable](https://github.com/puppetlabs/facter/blob/stable/lib/src/ruby/api.cc#L509), this should be `"ruby failed to run: %1%", output`.

File: ruby/src/api.cc [hunk 2]
```diff
@@ -469,7 +469,7 @@ namespace leatherman { namespace ruby {
                                                                   "break file if File.exist? file;"
                                                                   "false end)" });
         if (!success) {
-            LOG_WARNING("ruby failed to run", output, none);
+            LOG_WARNING("ruby failed to run: %1%", output);
```
Comment (MikaelSmith) [inline ruby/src/api.cc]: This used to `return library;` after the warning.

**Comment by MikaelSmith** [comment]
Ok, I think that's everything. @peterhuene?

**Comment by peterhuene** [comment]
:+1:

**Comment by MikaelSmith** [comment]
@Magisus can you bump the submodule in Facter#master?

**Pull request merged**